### PR TITLE
OSDOCS-3584: partial cluster update docs

### PR DESCRIPTION
**Pull request by opayne1**
[[OSDOCS-3584](https://issues.redhat.com/browse/OSDOCS-3584)]: Updates the EUS docs and updating cluster docs to reflect the new ability to do a partial update 

Version(s):
Applies to `enterprise-4.10` and `enterprise-4.11`

Link to docs preview:
1. [Preview link](https://deploy-preview-45783--osdocs.netlify.app/openshift-enterprise/latest/updating/preparing-eus-eus-upgrade.html) see the 3rd bullet.
2. [Preview link](https://deploy-preview-45783--osdocs.netlify.app/openshift-enterprise/latest/updating/updating-cluster-within-minor.html#prerequisites) see the 6th bullet.

Additional information:
Release notes can be found at #45730 and #11053

### CODE REVIEW

File: updating/updating-cluster-within-minor.adoc [hunk 1]
```diff
@@ -21,6 +21,8 @@ See xref:../authentication/using-rbac.adoc[Using RBAC to define and apply permis
 * Support for {op-system-base}7 workers is removed in {product-title} {product-version}. You must replace {op-system-base}7 workers with {op-system-base}8 or {op-system} workers before upgrading to {product-title} {product-version}. Red Hat does not support in-place {op-system-base}7 to {op-system-base}8 upgrades for {op-system-base} workers; those hosts must be replaced with a clean operating system install.
 * Ensure all Operators previously installed through Operator Lifecycle Manager (OLM) are updated to their latest version in their latest channel. Updating the Operators ensures they have a valid update path when the default OperatorHub catalogs switch from the current minor version to the next during a cluster update. See xref:../operators/admin/olm-upgrading-operators.adoc#olm-upgrading-operators[Upgrading installed Operators] for more information.
 * Ensure that all machine config pools (MCPs) are running and not paused. Nodes associated with a paused MCP are skipped during the update process. You can pause the MCPs if you are performing a canary rollout update strategy.
+//remove this???^ or maybe just add another bullet that you can break up the update?
```
Comment (opayne1) [inline updating/updating-cluster-within-minor.adoc:24]: Not going to remove. This was just a brainstorm. I have added another bullet.

**Comment by opayne1** [comment]
@rhamilto PTAL. The text is pretty similar to the release note. Added it to two places so people know they can also do a partial update. Thank you!

**Comment by mjpytlak** [comment]
@opayne1 peer review LGTM.

**Comment by opayne1** [comment]
@yapei This is for the partial upgrade that is going in with 4.10.14. Could I have your ack on this? Thank you!

**Comment by yanpzhan** [comment]
The update LG from QE side.

**Pull request merged**